equistore-core/tests/cpp/CMakeLists.txt

```diff
@@ -1,9 +1,10 @@
 cmake_minimum_required(VERSION 3.10)
 project(equistore-tests)
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_NO_SYSTEM_FROM_IMPORTED ON)
+if (${CMAKE_VERSION} VERSION_EQUAL 3.10)
```

Comment:
why is this needed? What if I have a higher cmake version is used? Don't you not need to set the CXX standard?

Reply:
We now set the C++ standard using target properties (which is only available since cmake 3.11). See `target_compile_features(catch PUBLIC cxx_std_11)` and similar for all targets

Reply:
Sorry, I though I was answering to a different PR ^^ This commit is actually part of #233, and will be replaced by the version in #267 once we merge it.